packages/mesh-apis/src/consumer_execute.rs

```diff
@@ -0,0 +1,9 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Uint128;
+
+#[cw_serde]
+pub enum ConsumerExecuteMsg {
+    MeshConsumerRecieveRewardsMsg {
+        rewards_by_validator: Vec<(String, Uint128)>,
+    },
+}
```

Comment:
Minor nit, I usually like keeping the `ExecuteMsg` in the relevant contracts... unless it's extensible like in `cw_nfts`.

Reply:
Sorry missed this.
The problem is that I need it in meta-staking, but consumer already imports meta-staking, so I can't import consumer in meta-staking (dependency loop), whats a better way of doing it?

I placed it in mesh-api because it seems like thats the role of it and we already do that with other msgs.

Reply:
Oh, that's fine then. Missed that. Ignore.